core/src/main/scala/org/broadinstitute/dsde/rawls/submissions/SubmissionsService.scala

```diff
@@ -538,14 +538,7 @@ class SubmissionsService(
 
     for {
       _ <- requireComputePermission(workspaceName)
-
```

Comment:
The changes here are primarily taking some validations that were scattered throughout the code, and consolidating them in `SubmissionRequestValidation`.